auth-web/src/views/auth/MaintainBusinessView.vue

```diff
@@ -29,6 +29,9 @@
           <v-btn v-else large color="#fcba19" @click="login()" class="my-5">
             Log in with BC Services Card
           </v-btn>
+          <p>New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
```

Comment:
if you want , you can consider doing the emit in html itself , rather than creating a dummy method for it..
v-on:click="$emit('eventname') syntax will work I guess.. upto you... I am not a fan of empty method to emit event alone..

Reply:
"separation of concerns"
It's good to keep the template focused on display, and use script to do the work.
(IMHO)